### DISCUSSION TITLE
Versionner la documentation avec Stylo ?

### DISCUSSION THREAD

**Discussion by thom4parisot** [EcrituresNumeriques/stylo#1050] [Ideas]
Et si on plaçait la documentation dans le répertoire `/docs` de Stylo ? Comme ça :
- on peut déployer la documentation avec la même version qui est déployée de Stylo (la prod en prod, la dev en dev)
- on peut préparer la documentation de la future version en prod
- on rappatrie la documentation sur le même nom de domaine (et on réduit la complexité technique de son hébergement)

Actuellement, il n'y a qu'une seule version de la/des documentation(s), qui tente de suivre les évolutions de Stylo.

**Comment by RochDLY** [comment]
Oui c'est une très bonne idée de faire les modifications dans le sens que tu proposes ! Il me semble que l'idée de la refonte de la documentation comprenait une partie des points que tu évoques (voir #845).
